### DISCUSSION TITLE
[HELP]: Align Subs using keybinds

### DISCUSSION THREAD

**Discussion by OmiChrome** [hooke007/mpv_PlayKit#287] [z_ENG_discuss]
i want to re-align or move subtitle in mpv , 
here is the Idea * press **`h,j,k,l`** like vim keybinds to move sub  
h = left 
j = down 
k = up 
l = left 

but how ? mpv manual doesn't seem to give such option or im wrong ? 

what i tried so so far 
```
# Move subtitles left
h sub-pos-x -= 10
# Move subtitles down
j sub-pos-y += 10
# Move subtitles up
k sub-pos-y -= 10
# Move subtitles right
l sub-pos-x += 10
```
but its seem to be depreciated

**Comment by OmiChrome** [comment]
this is what im using for now, however i want to desperately change it 
```
Ctrl+- add sub-pos -1                       # move subtitles up
Ctrl+= add sub-pos +1                       # move subtitles down
 ```

**Comment by hooke007** [comment]
You should post the requests to mpv, not here.

**Reply by OmiChrome** [reply]
😭  i was thinking its already possible ,
